client/dashboard/task-list/tasks/shipping/rates.js

```diff
@@ -8,14 +8,16 @@ import { Button, TextControl } from 'newspack-components';
 import classnames from 'classnames';
 import { Component, Fragment } from '@wordpress/element';
 import { FormToggle } from '@wordpress/components';
-import { get } from 'lodash';
 import PropTypes from 'prop-types';
 
 /**
  * WooCommerce dependencies
  */
 import { Flag, Form } from '@woocommerce/components';
 import { getCurrencyFormatString } from '@woocommerce/currency';
+import { CURRENCY } from '@woocommerce/wc-admin-settings';
+
+const { symbol = '$', symbolPosition = 'left' } = CURRENCY;
```

Comment:
Defaults are already defined in `client/settings/fallbacks.js`. Are they needed here too?

Reply:
Nope :)